doc/install.rst

```diff
@@ -0,0 +1,350 @@
+permissions to this file. You can create the shared memory file
+automatically by adding the following config file:
+
+``touch /etc/tmpfiles.d/10-looking-glass.conf``
+
+Add the following content to the file::
+
+   #Type Path Mode UID GID Age Argument
+
+   f /dev/shm/looking-glass 0660 user kvm -
+
+Be sure to set the UID to your local user.
+
+.. _looking_glass_service_windows:
+
+Looking Glass Service (Windows)
+-------------------------------
+
+You must first run the Windows VM with the changes noted above in either
+the :ref:`libvirt_configuration` or :ref:`qemu_commands` sections.
+
+.. _installing_the_ivshmem_driver:
+
+Installing the IVSHMEM Driver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Windows will not prompt for a driver for the IVSHMEM device, instead, it
+will use a default null (do nothing) driver for the device. To install
+the IVSHMEM driver you will need to go into the device manager and
+update the driver for the device "PCI standard RAM Controller" under the
+"System Devices" node.
+
+A signed Windows 10 driver can be obtained from Red Hat for this device
+from the below address:
+
+https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/upstream-virtio/
+
+Please note that you must obtain version 0.1.161 or later
+
+If the installation of the driver results in warnings or errors about
+driver signatures, ensure secure boot is turned off for the virtual
+machine bios/uefi.
+
+.. _a_note_about_ivshmem_and_scream_audio:
+
+A note about IVSHMEM and Scream Audio
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+   Using IVSHMEM with Scream may interfere with Looking Glass, as they may try
+   to use the same device.
+
+Please do not use the IVSHMEM plugin for Scream.
+Use the default network transfer method. The IVSHMEM method induces
+additional latency that is built into its implementation. When using
+VirtIO for a network device the VM is already using a highly optimized
+memory copy anyway so there is no need to make another one.
+
+If you insist on using IVSHMEM for Scream—despite its inferiority to the
+default network implementation—the Windows Host Application can be told
+what device to use. Create a ``looking-glass-host.ini`` file in the same
+directory as the looking-glass-host.exe file. In it, you can use the
+``os:shmDevice`` option like so:
+
+.. code:: INI
+
+   [os]
+   shmDevice=1
+
+.. _using_the_windows_host_application:
+
+Using the Windows Host Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start by downloading the correct version for your release from
+https://looking-glass.io/downloads. You can either choose between
+**Official Releases**, which are stable; or **Release Candidates**, new versions
+about to be stable, but haven't passed validation.
+
+.. note::
+   If your **looking-glass-client** was created by building from the **master
+   branch** you have to pick the **Bleeding Edge** version.
+
+Next, use `7-Zip <7-zip.org/>`_ to extract the zip archive using the commit
```

Comment:
I think we now use the old PKZIP encryption algorithm and 7-Zip is no longer needed.